player/public/coreplugins/org.ekstep.pdfrenderer-1.0/renderer/plugin.js

```diff
@@ -356,18 +370,21 @@ org.ekstep.contentrenderer.baseLauncher.extend({
                         if($("#pdf-meta").hasClass("higheropacity"))
                             instance.applyOpacityToNavbar(false);
                     });
-
+                    setTimeout(function () {
```

Comment:
This can be added in the above 
instance.headerTimer = setTimeout(function() {
  isPageRenderingInProgress = false;
})

You can remove this addition setTimeout function